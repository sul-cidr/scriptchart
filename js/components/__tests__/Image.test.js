
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import Image from '../Image';


describe('Image test', () => {
  it('Image should match snapshot', () => {
    const component = renderer.create(<Image
       />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
