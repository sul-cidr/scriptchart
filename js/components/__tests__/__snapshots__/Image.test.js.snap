// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image test Image should match snapshot 1`] = `
<div
  style={
    Object {
      "backgroundColor": "gray",
      "backgroundImage": "url(\\"undefined\\")",
      "backgroundPosition": "center center",
      "backgroundRepeat": "no-repeat",
      "backgroundSize": "contain",
      "height": 100,
      "width": 100,
    }
  }
/>
`;
